doc/manual/command-ref/conf-file.xml

```diff
@@ -430,6 +430,21 @@ flag, e.g. <literal>--option gc-keep-outputs false</literal>.</para>
   </varlistentry>
 
 
+  <varlistentry><term><literal>netrc-file</literal></term>
+
+    <listitem><para>If set to an absolute path to a <filename>netrc</filename>
```

Comment:
I know you explain the format below, but might be nice to link to a spec (if one exists?) too